webpack.config.js

```diff
@@ -19,10 +19,29 @@ module.exports = [
     resolve: {
       alias: {
         fs: require.resolve("./src/vfs"),
-        module: require.resolve("./src/mocks/dummy")
+        "graceful-fs": require.resolve("./src/vfs"),
+        module: require.resolve("./src/mocks/dummy"),
+        chokidar: require.resolve("./src/mocks/chokidar"),
+        "uglify-js": require.resolve("./src/mocks/dummy"),
+        // These modules are used by virtualfs to fake async fs calls
+        "core-js/library/fn/set-immediate": require.resolve(
```

Comment:
How does that work with running the benchmark suite via `node`?

Reply:
See below